Kickstarter-iOS/Library/UIImageView+URL.swift

```diff
@@ -15,3 +18,30 @@ extension UIImageView {
                      completion: nil)
   }
 }
+
+private enum Associations {
+  fileprivate static var imageUrl = 0
+}
+
+extension Rac where Object: UIImageView {
```

Comment:
don't we ideally want all images in the app to have a cross-dissolve though? (i assumed we'd change all `af_setImageWithURL` to `ksr_setImageWithURL` at some point.)